templates/openstack/README.md

```diff
@@ -0,0 +1,30 @@
+Monitoring Openstack
+====================
+
+Monitoring Openstack environment using the Openstack Command Line client.
+
+* Images
+* Ports
+* Resource Limits
+* Server List
+* Resource Usage
+* Volume List
+* Zone Availability
+
+The monitoring requires the Openstack CLI to be installed on the (LINUX) server where the Patrol Agent and the Monitoring Studio KM is installed.
+
+https://docs.openstack.org/newton/user-guide/common/cli-install-openstack-command-line-clients.html
+
+** Set up **
+
+* Create a user account for monitoring.
+
+* Set the Environment Variables: https://docs.openstack.org/newton/user-guide/common/cli-set-environment-variables-using-openstack-rc.html
+ An Openstack RC file is embedded in the template. Edit the below lines in the embedded "openrc-demo-monitor.sh" with appropriate values.
+
+export OS_TENANT_ID="<TENANT_ID>"
+export OS_TENANT_NAME="<TENANT_NAME>"
+export OS_USERNAME="<USER_NAME>"
+
+* The password is required as a parameter. To workaround this prompt,set the password to the env variable $OS_PASSWORD_INPUT.
```

Comment:
I still do not understand, I'm sorry! 😅

Does this environment variable need to be set in the **openrc-demo-monitor.sh** file? In which case, just add it to the above variables (with `OS_USERNAME`).

Reply:
From what I understood, please update as below:

````md
Set up
------

* Create a user account for the monitoring.

An Openstack RC script file is embedded in the template. Use the *Studio* tab to edit the below lines in the **openrc-demo-monitor.sh** embedded file with appropriate values:

```bash
export OS_TENANT_ID="<TENANT_ID>"
export OS_TENANT_NAME="<TENANT_NAME>"
export OS_USERNAME="<USER_NAME>"
export OS_PASSWORD="<PASSWORD>"
```

The password is required as a parameter. As an alternative to putting the password in clear in the embedded file, you can set the `$OS_PASSWORD` variable in the environment of the PatrolAgent process.
````

Another question: Why don't we use macros in the embedded file itself? Doesn't it work?